src/components/RedirectByName.tsx

```diff
@@ -11,15 +11,27 @@ export const RedirectByName = () => {
   const [address, setAddress] = useState('')
   const { selectNetwork } = useNetwork()
 
+  const [actNetwork, setActualNetwork] = useState<string>()
```

Comment:
Just a nit on the name
```suggestion
  const [redirectionNetwork, setRedirectionNetwork] = useState('')
```